pandas/io/parsers/readers.py

```diff
@@ -382,6 +411,7 @@
     "memory_map": False,
     "error_bad_lines": True,
```

Comment:
the original options need to be defaulted to None, then you can tell if they are set.